app/src/main/java/org/p2p/wallet/common/storage/FilesDirStorageRepository.kt

```diff
@@ -0,0 +1,51 @@
+package org.p2p.wallet.common.storage
+
+import android.content.Context
+import com.google.gson.Gson
+import timber.log.Timber
+import java.io.IOException
+import kotlinx.coroutines.withContext
+import org.p2p.core.dispatchers.DefaultDispatchers
+
+class FilesDirStorageRepository(
+    private val context: Context,
+    private val gson: Gson,
+    private val dispatchers: DefaultDispatchers
+) : ExternalStorageRepository {
+
+    // TODO: use toJson with streamed JsonReader instead of raw string
```

Comment:
do we need a task for this or just leave this todo like this?
@1sakov

Reply:
just leave, it's low priority